packages/core/src/docdb/explorer/dbClusterNode.ts

```diff
@@ -49,6 +64,22 @@ export class DBClusterNode extends AWSTreeNodeBase implements AWSResourceNode {
         })
     }
 
+    private getContext(): DBClusterNodeContext {
+        if (this.status === 'available') {
+            return DBClusterRunningContext
+        } else if (this.status === 'stopped') {
+            return DBClusterStoppedContext
+        }
+        return DBClusterPendingContext
+    }
+
+    public getDescription(): string | boolean {
```

Comment:
what will `getDescription()` be used for?

Reply:
It's called from the constructor to assign the text that is displayed to the right of the name in a muted color.